src/main/java/jp/ikedam/jenkins/plugins/extensible_choice_parameter/TextareaChoiceListProvider.java

```diff
@@ -143,6 +143,21 @@ public String getDefaultChoice()
         return defaultChoice;
     }
     
+
+    private boolean addToTop;
+
+    /**
+     * Returns if user-added values should be added to the top of the choice list.
+     * 
+     * @return if user-added values should be added to the top of the choice list.
+     * @see jp.ikedam.jenkins.plugins.extensible_choice_parameter.ChoiceListProvider#isAddToTop()
+     */
+    @Override
+    public boolean isAddToTop()
+    {
+        return addToTop;
+    }
+    

```

Comment:
Move to `AddEditedChoiceListProvider`.
This should be treated like `whenToAdd`.